### PR TITLE
fix(refs DPLAN-11601): Only allow orgas with E-Mail addresses to be ivitable

### PR DESCRIPTION
**Pull request by salisdemos**
### Ticket
DPLAN-11601

Orgas with no Email-address (double opt in pending) should be unselectable in the "invite orgas list".
Add "lock-checkbox-by" to the invite institution list by email, to prevent that instutions without an email adress can be invited , wich would trigger a mailsend to an undefined  address. 

To archive this the main change had to happen in demosplan-ui. See related PRs

### How to review/test

with the changes from demosplan-ui:
on the page verfahren/<procedure-id>/einstellungen/benutzer/hinzufuegen

only orgas with an email address should be selectable. (should also work with the select all toggle) 

### Linked PRs (optional)
List other PRs that are somehow connected to this and explain the connection.

- Other [PR1 #{PR-number1}](https://github.com/demos-europe/demosplan-ui/pull/878)

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [ ] Move the tickets on the board accordingly
- [ ] Data-Cy attributes added/updated ([conventions](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/twig_html.html#guideline-for-naming-cypress-hooks))

### CODE REVIEW

File: client/js/components/procedure/admin/DpAddOrganisationList.vue [hunk 1]
```diff
@@ -178,6 +179,7 @@ export default {
       return Object.values(this.invitableToebItems).reduce((acc, item) => {
         const locationContactId = item.relationships.locationContacts.data[0].id
         const locationContact = this.getLocationContactById(locationContactId)
+        const hasNoEmail = item.attributes.participationFeedbackEmailAddress === null
```
Comment (gruenbergerdemos) [inline client/js/components/procedure/admin/DpAddOrganisationList.vue:182]: Could this also be empty string or undefined?

```suggestion
        const hasNoEmail = !item.attributes.participationFeedbackEmailAddress
```

**Review by gruenbergerdemos — CHANGES_REQUESTED** [review]
Hint for locked items is missing. It can be done via translations prop (see https://github.com/demos-europe/demosplan-core/pull/3167/files)

**Comment by salisdemos** [comment]
> Hint for locked items is missing. It can be done via translations prop (see https://github.com/demos-europe/demosplan-core/pull/3167/files)

True. I just forgot to implement that.

**Comment by salisdemos** [comment]
> Hint for locked items is missing. It can be done via translations prop (see https://github.com/demos-europe/demosplan-core/pull/3167/files)

True. I just forgot to implement that.

**Review by gruenbergerdemos — APPROVED** [review]

**Pull request merged**